lib/mongo.js

```diff
@@ -0,0 +1,125 @@
+const mongo = require('./mongo_conn');
+const { ObjectID } = require('mongodb');
+const CONFIG = mongo.config;
+const LIMIT = 20;
+
+class Mongo {
+
+    constructor(table, dbName = CONFIG.AMK_MONGO_DEFAULT_DATABASE) {
+        this.TABLE = table;
```

Comment:
Is there any reason you use upper case here? and why use table instead of collection since mongodb is NoSQL.

Reply:
yes U are right.

Reply:
done.